frontend/packages/ceph-storage-plugin/src/components/bucket-class/wizard-pages/namespace-store-pages/multi-namespace-store.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, Flex, FlexItem, Form, FormGroup, Stack, Title } from '@patternfly/react-core';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { NamespaceStoreKind } from '../../../../types';
+import { NamespaceStoreList } from '../../../namespace-store/namespace-store-table';
+import { Action, State } from '../../state';
+import NamespaceStoreModal from '../../../namespace-store/namespace-store-modal';
+import { NamespaceStoreDropdown } from '../../../namespace-store/namespace-store-dropdown';
+
+export const MultiNamespaceStorePage: React.FC<MultiNamespaceStoreProps> = React.memo(
+  ({ state, dispatch, namespace }) => {
+    const { t } = useTranslation();
+    const openModal = () => NamespaceStoreModal({ namespace });
+    const [selectedCount, setSelectedCount] = React.useState(state.readNamespaceStore.length);
+    const onSelectNamespaceStoreTable = (selectedNamespaceStore: NamespaceStoreKind[]) => {
+      dispatch({ type: 'setReadNamespaceStore', value: [...selectedNamespaceStore] });
+      setSelectedCount(selectedNamespaceStore.length);
+    };
+
+    const onSelectNamespaceStoreDropdown = (selectedNamespaceStore: NamespaceStoreKind) => {
+      dispatch({ type: 'setWriteNamespaceStore', value: [selectedNamespaceStore] });
+    };
+    return (
+      <div className="nb-create-bc-step-page">
+        <Flex
+          justifyContent={{ default: 'justifyContentSpaceBetween' }}
+          alignItems={{ default: 'alignItemsCenter' }}
+        >
+          <Stack>
+            <Title size="xl" headingLevel="h2">
+              {t('ceph-storage-plugin~Read NamespaceStores')}
+            </Title>
+            <p className="nb-create-bc-step-page-form__element--light-text">
+              {t(
+                'ceph-storage-plugin~Select list of namespace stores, defines the read targets of the namespace bucket',
+              )}
+            </p>
+          </Stack>
```

Comment:
Try to stick to one grid layout - flex/stack

Reply:
changed stack to a flex item